app-benchmarks/uefi-sct/spec

```diff
@@ -0,0 +1,4 @@
+VER=202406
+SRCS="git::copy-repo=true;commit=tags/edk2-test-stable${VER}::https://github.com/tianocore/edk2-test.git"
```

Comment:
For my note on sources above:

```
UPSTREAM_VER=202406
EDK2_VER=202405
VER=${UPSTREAM_VER}+edk2${EDK2_VER}
SRCS="git::commit=tags/$UPSTREAM_VER;rename=uefi-sct::https://github.com/tianocore/edk2-test.git \
      git::commit=tags/$EDK2_VER;rename=edk2::https://github.com/tianocore/edk2.git"
CHKSUMS="SKIP \
         SKIP"
```

Now, both of them will be cloned under the same directory, you will now need to specify one of the repos as the main source (corresponding to `$SRCDIR`). For instance:

```
SUBDIR=uefi-sct
```

And `edk2` will now be at `"$SRCDIR"/../edk2` - adjust your scripts accordingly.